gl-pvr.c

```diff
@@ -96,8 +96,8 @@ static inline void pvr_hdr_submit(const GLuint *src) {
 }
 #else
 
-#define pvr_list_submit(src, n) sq_cpy(TA_SQ_ADDR, (src), ((n) << 5))
-#define pvr_hdr_submit(src) sq_cpy(TA_SQ_ADDR, (src), 32)
+#define pvr_list_submit(src, n) sq_cpy((void *)PVR_TA_INPUT, (src), ((n) << 5))
+#define pvr_hdr_submit(src) sq_cpy((void *)PVR_TA_INPUT, (src), 32)
```

Comment:
You can use pvr_sq_cpy with TA

Reply:
I think this is what you want to use:

pvr_sq_load((void *)0, src, size, PVR_DMA_TA);